modules/nf-scil/betcrop/fslbetcrop/meta.yml

```diff
@@ -0,0 +1,59 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/yaml-schema.json
+name: "betcrop_fslbetcrop"
+description: Perform Brain extraction using FSL BET followed by cropping empty planes around the data.
+keywords:
+  - FSL
+  - BET
+  - Crop
+tools:
+  - "fslbetcrop":
```

Comment:
In this section, if you want to descbribe each too, create a new sub-section :
```
- "FSL"
      description: ...
- "Mrtrix"
      description: ...
- "Scilpy"
      description: ...
```

Do it if you really want to, but I won't enforce a complete documentation here. Else, just keep scilpy

Reply:
That's a good idea! For some reasons I thought we could only place one entry...